decidim-assemblies/app/views/decidim/assemblies/admin/assemblies/index.html.erb

```diff
@@ -77,15 +77,11 @@
                <% else %>
                  <span class="action-space icon"></span>
                <% end %>
-               <% if assembly.children.count.positive? || allowed_to?(:create, :assembly) %>
-                 <%= icon_link_to "government-line",
-                                  url_for(query_params_with(parent_id_eq: assembly.id)),
-                                  t("decidim.admin.titles.assemblies"),
-                                  class: "action-icon--dial #{"highlighted" if assembly.children.count.positive?}" %>
-               <% else %>
-                 <span class="action-space icon"></span>
-               <% end %>
-              <% if allowed_to? :copy, :assembly, assembly: assembly %>
+               <%= icon_link_to "government-line",
+                                url_for(query_params_with(parent_id_eq: assembly.id)),
+                                t("decidim.admin.titles.assemblies"),
+                                class: "action-icon--dial #{"highlighted" if assembly.children.count.positive?}" %>
```

Comment:
@renato
Given Assembly "A" with no sub-assemblies. 
Given I am Assembly Admin for Assembly "A". 
Given I go in assemblies list and I see Assembly A.
Then, I should see the "assemblies" button to go and create sub-assemblies. 

=> I am almost sure this test fails.

Reply:
@froger
This test works :) 

https://github.com/user-attachments/assets/34908eb7-d657-4b61-874e-f6b05f724e65